api-gateway/src/main/resources/error.html

```diff
@@ -10,7 +10,7 @@
 
     <title>SAVE Cloud Service</title>
 
-    <link rel="stylesheet" type="text/css" href="main.css">
+    <link rel="stylesheet" type="text/css" href="/main.css">
```

Comment:
Ah, that was the reason, lol

Reply:
The main reason for a problem was some plugin for HTML processing. It used to add a script tat to the head of html. The path was relative there.